arma3/@task_force_radio/addons/task_force_radio/functions/fn_processPlayerPositions.sqf

```diff
@@ -27,7 +27,9 @@ if !(isNull (findDisplay 46)) then {
 				tf_nearPlayers = call TFAR_fnc_getNearPlayers;
 			};
 
-			_other_units = allUnits - tf_nearPlayers;
+			_other_units = allUnits - allPlayers; // non player units
+			_other_units = _other_units + allPlayers; // add player units
```

Comment:
to prevent duplicates,
allUnits + allPlayers could returns some players two times